storyblok/storyblok-design-system
# refactor(button): render api to template syntax

## Pull request type

Jira Link: -




## How to test this PR


This refactors the component to the template syntax. Checkout out this branch in the storyfront and see how it renders with the app- 

## What is the new behavior?



- refactoring of the button
- 
- 

## Other information
